cypress/Pages/Login/index.js

```diff
@@ -0,0 +1,46 @@
+import Page from "../page";
+import { Selectors } from "../../../src/components/Login/selectors";
+
+class LoginPage extends Page {
+  get emailInput() {
```

Comment:
Taking advantage of the read selectors here [data-test="..."] from the selectors file colocated next to the component